apimodels/agent_models.go

```diff
@@ -183,6 +183,10 @@ type RegistrySettings struct {
 	Password string `mapstructure:"registry_password" json:"registry_password" yaml:"registry_password"`
 }
 
+type InstallationToken struct {
+	Token string `bson:"token" json:"token"`
```

Comment:
Is the BSON tag needed? Seems like it shouldn't need the BSON tags, unless it's stored in the DB.

Reply:
Seems like the tag is still here.